app/_includes/md/mesh/install-universal-run.md

```diff
@@ -36,8 +29,6 @@ in `/usr/local/bin/` by executing:
 $ ln -s ./kumactl /usr/local/bin/kumactl
 ```
 
-<div class="alert alert-ee blue">
 <strong>Note:</strong> By default, this will run {{site.mesh_product_name}} with a memory
```

Comment:
With the <div> tag removed, these are all still labelled as notes but are not using Note styling. Is that something we want to do? Might be better to be more discerning for whether something is a note or not, and always use the blockquote style if it is one. Plus, if you don't want too many panels on a page, then it makes you start thinking about why you're adding so many notes, and whether they're actually better as part of the content flow. 

Tl;dr, I suggest removing the "Note" prefix in these sections that you've de-blued. The final solution might be to rework these topics altogether, but I realize that you're not trying to do that here.

(Applies to all changes in this PR)